### PR TITLE
magento/pwa-studio#: Fix links in the “Contributing” section

### PR DESCRIPTION
**Pull request by atwixfirster**
## Description
This pull request fixes links in the "Contributing" section of https://github.com/magento/pwa-studio/tree/develop/pwa-devdocs page.

## Related Issue

Closes #ISSUE_NUMBER.

## Verification Steps

1. Go to https://github.com/magento/pwa-studio/tree/develop/pwa-devdocs page.
2. Check `contribution guidelines` and `code of conduct` links.

**Actual result:** mentioned links go to `404` page.
**Expected result:** links should contain correct URLs

## Proposed Labels for Change Type/Package

- [ ] major (e.g x.0.0 - a breaking change)
- [ ] minor (e.g 0.x.0 - a backwards compatible addition)
- [ ] patch (e.g 0.0.x - a bug fix)

## Checklist:

- [ ] I have updated the documentation accordingly, if necessary.
- [ ] I have added tests to cover my changes, if necessary.

Thank you!

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1557) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/pwa-studio?pullRequest=1557) before we can accept your contribution.<br/>

**Comment by atwixfirster** [comment]
> The proper fix for this would be to just update line 79 and 80 with the correct URL.
> 
> Also, I'm not sure why the link definitions on the bottom of this page were changed to actual links, but it breaks the reference-style links used on the page.

Hi, @jcalcaben !

Your suggestions have been implemented.

Could you please review? Thanks

File: pwa-devdocs/README.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ If you want to contribute, please review the [contribution guidelines] and follo
 
 This project is licensed under OSL-3.0 - see the [LICENSE.txt] file for details.
 
-[contribution guidelines]: ../../.github/CONTRIBUTING.md
-[code of conduct]: ../../.github/CODE_OF_CONDUCT.md
+[contribution guidelines]: https://magento.github.io/pwa-studio/technologies/contribute/
+[code of conduct]: https://github.com/magento/pwa-studio/blob/master/.github/CODE_OF_CONDUCT.md
```
Comment (sirugh) [inline pwa-devdocs/README.md]:
```suggestion
[code of conduct]: https://github.com/magento/pwa-studio/blob/develop/.github/CODE_OF_CONDUCT.md
```

Comment (atwixfirster) [reply pwa-devdocs/README.md]: **fixed**

Comment (atwixfirster) [reply pwa-devdocs/README.md]: @sirugh , could you please review?

Thank you!

**Comment by atwixfirster** [comment]
@sirugh , could you please add "partner-contributions" and "Partner Atwix" labels to this PR?

Thank you!

**Pull request merged**